node/actors/network/src/rpc/mod.rs

```diff
@@ -26,6 +26,7 @@ pub(crate) mod get_block;
 mod metrics;
 pub(crate) mod ping;
 pub(crate) mod push_block_store_state;
+pub(crate) mod push_signature;
```

Comment:
push_l1batch_signature

Reply:
I changed it to `push_batch_signature` to mantain consistency and go with just `batch` instead of `l1batch`